spec/support/methods/feature_test_helper_methods.rb

```diff
@@ -0,0 +1,45 @@
+module FeatureTestHelperMethods
+  def start(the_flow:, at:)
+    visit "/#{at}"
+    ensure_page_has(header: the_flow)
+    click_govuk_start_button
```

Comment:
I think we should get rid of the method `click_govuk_start_button`, because I don't think we will use it anywhere else and its a lot of overhead (separate helper file and module) just for this simple method
```suggestion
    click_link "Start now"
```

Reply:
Yup, agree - will remove and update the `start` method.